toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -34,14 +34,7 @@ <h3 class="categories-heading">{{ parent_category.label }}</h3>
                 last=loop.last,
                 hint=None
               %}
-
-                {# TODO: PLS FIX #}
-                {% if PAGES_BUILDER %}
-                  {% include 'forms/_selection-button.html' %}
-                {% else %}
-                  {% include 'toolkit/forms/_selection-button.html' %}
-                {% endif %}
-
+                {% include 'toolkit/forms/_selection-button.html' %}
               {% endwith %}
```

Comment:
So why do you have this at the top of the pull request?

<img width="693" alt="screen shot 2017-05-12 at 10 02 00" src="https://cloud.githubusercontent.com/assets/11633362/25991155/3cac753c-36fa-11e7-9e56-9e389f4f8fab.png">